src/mem/port.hh

```diff
@@ -373,52 +383,52 @@ class SlavePort : public Port, public AtomicResponseProtocol,
     sendTimingSnoopReq(PacketPtr pkt)
     {
         try {
-            TimingResponseProtocol::sendSnoopReq(_masterPort, pkt);
+            TimingResponseProtocol::sendSnoopReq(_requestPort, pkt);
         } catch (UnboundPortException) {
             reportUnbound();
         }
     }
 
     /**
-     * Send a retry to the master port that previously attempted a
-     * sendTimingReq to this slave port and failed.
+     * Send a retry to the requestor port that previously attempted a
+     * sendTimingReq to this response port and failed.
      */
     void
     sendRetryReq()
     {
         try {
-            TimingResponseProtocol::sendRetryReq(_masterPort);
+            TimingResponseProtocol::sendRetryReq(_requestPort);
         } catch (UnboundPortException) {
             reportUnbound();
         }
     }
 
     /**
-     * Send a retry to the master port that previously attempted a
-     * sendTimingSnoopResp to this slave port and failed.
+     * Send a retry to the requestor port that previously attempted a
+     * sendTimingSnoopResp to this response port and failed.
      */
     void
     sendRetrySnoopResp()
     {
         try {
-            TimingResponseProtocol::sendRetrySnoopResp(_masterPort);
+            TimingResponseProtocol::sendRetrySnoopResp(_requestPort);
         } catch (UnboundPortException) {
             reportUnbound();
         }
     }
 
   protected:
     /**
-     * Called by the master port to unbind. Should never be called
+     * Called by the requestor port to unbind. Should never be called
      * directly.
      */
     void slaveUnbind();
```

Comment:
There were two functions (slaveUnbind() and slaveBind()) we were unsure how to deprecate. In the RequestPort class should this now be RequestPort::requestorUnbind() and in the SlavePort class we'd need SlavePort::slaveUnbind(). Both of which would need to be identical functions with the same contents just a different name that we need to define in the .cc? These were the only two cases we've seen so far where the old terminology effected function names in this class.

Reply:
You should be able to just change the names of these functions. This shouldn't be an exposed API, and we should be able to change it however we feel.

Reply:
Wouldn't some references to the oldAPI we've yet to revise still call SlavePort::slaveUnbind()?

Reply:
Actually looking at it closer, it looks like slaveUnbind() is only called in token_port.cc and slaveBind() is never called. In that case I think we could just change the name as you suggested and update the only reference. The description on the function says  "Called by the requestor port to unbind. Should never be called directly." which seems odd since we only found one reference and for only one of the functions?